### DISCUSSION TITLE
Device not Supportet but in the List

### DISCUSSION THREAD

**Discussion by peterossege** [Koenkk/zigbee2mqtt#16651] [Devices]
When I try to paar "Livarno Home outdoor spotlight", I get the message that it is not supported but it is in the supported list.

https://www.zigbee2mqtt.io/devices/HG08010.html

I have the same issue with several other devices.

**Comment by Danit2** [comment]
Are you up to date with your version from zigbee2mqtt?

**Reply by peterossege** [reply]
Yes, I am! I do so:

# Stop Zigbee2MQTT and go to directory
sudo systemctl stop zigbee2mqtt
cd /opt/zigbee2mqtt

# Backup configuration
cp -R data data-backup

# Update
git checkout HEAD -- npm-shrinkwrap.json
git pull
npm ci

# Restore configuration
cp -R data-backup/* data
rm -rf data-backup

# Start Zigbee2MQTT
sudo systemctl start zigbee2mqtt

**Reply by Danit2** [reply]
On which version are you? 
1.30.1 ?

Show your database entry of the device?

**Comment by peterossege** [comment]
Did I make anything wrong with my update?

**Reply by peterossege** [reply]
I switched to home assistant with zigbee integration - everything works fine ;-)
